amplify/backend/function/deleteUser/src/index.py

```diff
@@ -10,23 +10,23 @@
 db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
 
 def lambda_handler(event, context):
-    username = event["pathParameters"]["userId"]
+    userID = event["pathParameters"]["userId"]
 
      # Get vertex ID
-    query = f"g.V().has('username', '{username}').id();"
+    query = f"g.V().has('id', '{userID}').id();"
```

Comment:
Nit for consistency: In `/function/getUser` you also query based on the label: `g.V().hasLabel('user').has('id', '{userID}')`. Is that necessary here?

Reply:
likely not, if we ever decide to add nodes with a label other than user that also has an ID property then yes. But we probably don't need it

Reply:
I'll add it for consistency and future proofing anyway. It doesn't hurt to have it.